docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+---
+draft: False
+date: 2024-01-08
+slug: what-i-learned-from-trying
+categories:
+  - Personal
+authors:
+  - jxnl
+---
+
+# What I Learned from Trying
+
+After writing my post [advice for young people](./advice.md), a couple of people asked about my learning process. I could discuss overcoming plateaus or developing mastery, learning for the joy of learning. I could also talk about how to avoid feeling overwhelmed by new topics and break them down into smaller pieces. However, I think that has been done before.
+
+Instead, I'm going to explore a new style. I'm just going to go through a chronological telling of my life and what i learned about learning.
+
+I'm going to break this down into two sections:
+
+**What I did in high school and college**
+
+1. Design
+2. Data Science
+
+**What I did after college**
+
+- Pottery
+
+- Weightlifting
+
+- Jiu Jitsu
+
+- Rocket League
+
+It's a false dichotomy in the sense that the tactics and strategies for knowledge acquisition apply to physical skills, and the volume and leverage required to learn physical skills also apply to knowledge acquisition. But consider this an experiment, I'll go over each one and a lesson I learned.
+
+<!-- more -->
+
+## Environments Matter
+
+When I was a kid (~ 10 years old), both my mom and dad worked two jobs, and no one was at home. But every Saturday, my mom would take the time to walk me to the library, where I would borrow 10 books. I was only allowed to use the computer for an hour a day and watch 30 minutes of television per week, basically just one cartoon. Learning became my escape. I wanted to acquire knowledge and understand physics, such as igneous rocks and metamorphic ones. I also learned about different types of dinosaurs and animals.
+
+My parents were not around but i know the one thing they showed me was that education was the way out out poverty, and that they worked very hard to make sure i had access to it. This is all to say that I grew up in a place with very few distractions, and it was very easy to learn.
+
+## Don't study things you'll be taught
+
+This next few section will mostly be random things. I had noticed about my learning through high school and university.
```

Comment:
These next few sections will mostly be random things I had noticed about my learning through high school and university.